Pages/questionPage/basicGardening/question1.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="https://flora-ui.netlify.app/src/app.css">
+    <link rel="stylesheet" href="./basicGardening.css">
+    <title>FloraQuiz | MCQs</title>
+</head>
+<body>
+    <nav class="flex-r">
+        <a class="m1 p1 m-right-auto text-decor-none" href="../../../index.html"><span class=" app-name">floraQuiz<i class="fa-solid fa-question"></i></span></a>
+        <div class="m1 p1 align-center">
+            <a href="../../../index.html" class="btn btn-link text-lg text-decor-none"><i class="fa-solid fa-house-chimney"></i>Home</a>
+            <a href="" class="btn btn-link text-lg text-decor-none"><i class="fa-solid fa-right-to-bracket"></i>Login</a>
+        </div>
+    </nav>
+
+    <main class="flex-c align-center">
+        <h3 class="m2-top-bottom">Basic Gardening Quiz</h3>
+
+        <div class="p2 question-container">
+
+        <div class="question-header flex-r">
+            <p class="text-xlg">Question: 1/5</p>
+            <p class="text-xlg">Score: 0</p>
+        </div>
+
+        <p class="question text-xlg text-bold-weight">Which of these is a great source of nitrogen for your soil?</p>
+
+        <ul class="list-non-bullet">
+            <li tabindex="1" class="question-option text-lg text-center">Coffee grounds</li>
+            <li tabindex="1" class="question-option text-lg text-center">Sugar</li>
+            <li tabindex="1" class="question-option text-lg text-center">Banana peels</li>
+            <li tabindex="1" class="question-option text-lg text-center">Eggs</li>
+        </ul>
+
+        <div class="flex-c align-center">
+            <a href="./question2.html"><button class="m1 btn btn-secondary">Next Question</button></a>
+        </div>
+
+        </div>
+    </main>
+
+    <footer class="footer">
+        <p>Made with <i class="red-color fas fa-heart"></i> by Vrinda Mahajan </p>
+        <ul class="m2 list-non-bullet">
+            <li class="p1 inline"><a class="footer-icons" target="_blank" href="https://github.com/vrinda-mahajan"><i class="fab fa-github"></i></a></li>
+            <li class="p1 inline"><a class="footer-icons" target="_blank" href="https://twitter.com/vrinda_mahajan"><i class="fab fa-twitter"></i></a></li>
+            <li class="p1 inline"><a class="footer-icons" target="_blank" href="https://www.linkedin.com/in/vrinda-mahajan-7a2b2a20b/"><i class="fab fa-linkedin-in"></i></a></li>
+        </ul>
+        <div>© copyright 2022 Flora UI</div>
+    </footer>
+</body>
+</html>
```

Comment:
just a nitpick but adding a newline at the end of files is considered a good practice
https://thoughtbot.com/blog/no-newline-at-end-of-file